### ISSUE TITLE
A problem about miou on iSAID and potsdam

### ISSUE THREAD

**Issue by SecretplayeRava** [ZaiyiHu/CTFA#1]
I downloaded the processed datasets and codes, and trained it on a 3090 GPU. On the iSAID dataset, the best mIoU is 31.5, and on Potsdam it is 38.6, which is a significant gap compared to the ckpts sprovided by the authors. The training logs are as follows. Could you please help me identify where the issue might be?
[iSAID_train.log](https://github.com/user-attachments/files/18225867/iSAID_train.log)
[potsdam_train.log](https://github.com/user-attachments/files/18225869/potsdam_train.log)

**Comment by ZaiyiHu** [comment]
Sorry for the performance issues caused. I'm not quite sure why there is such a deviation... The learning rate in the log seems normal, but this point has also decreased compared to the original ToCo, and all losses appear to be decreasing normally. Perhaps you should try fine-tuning the loss weight of the foreground activation branch or the learning rate of additional decoders to further optimize.
